lib/commons/builder/queries/legislative_index_terms.rq.liquid

```diff
@@ -26,6 +27,31 @@ WHERE {
                                    pq:P2937 ?term ] .
   }
 
+  OPTIONAL {
+    ?house p:P1342 ?numberOfSeatsHouseStatement .
+    ?numberOfSeatsHouseStatement ps:P1342 ?numberOfSeatsHouse .
+    OPTIONAL { ?term (wdt:P580|wdt:P571) ?termStart }
+    OPTIONAL { ?term (wdt:P582|wdt:P576) ?termEnd }
```

Comment:
Since there are some terms that have both inception and start date, I think it might be worth explicitly picking one using coalesce, to avoid multiple results (as done in https://github.com/mysociety/verification-pages/pull/240/commits/618f39e2ed60c56539fd960de2a798c46c1c4217 recently perhaps)

Reply:
Ah, yes. Good point, and I like the solution. Will do this too. Will also apply it higher up in the query too as a separate commit. These ?termStart/?termEnd bindings are duplicated because the higher-up ones aren't visible to the FILTER in the lower-down OPTIONAL.